ee/experimental/incident_utils.py

```diff
@@ -145,4 +287,57 @@ def shape_incidents(alerts: pd.DataFrame, unique_alert_identifier: str, incident
                 'alert_fingerprints': local_alerts[unique_alert_identifier].unique().tolist(),
             })
 
-    return incidents
\ No newline at end of file
+    return incidents
+
+
+def generate_incident_summary(incident: Incident, use_n_alerts_for_summary: int = -1) -> str:
+    if "OPENAI_API_KEY" not in os.environ:
+        logger.error("OpenAI API key is not set. Incident summary generation is not available.")
```

Comment:
is it an error?

Reply:
I suppose, summary generation is quite important. Do you think it should be downgraded to logger.warning?